content/usgs-packages/geoknife_Discovery.Rmd

```diff
@@ -0,0 +1,233 @@
+
+```{r explore-fabric, message=FALSE, warning=FALSE}
+# no variables defined yet
+variables(evap_fabric)
+
+# find what variables are available
+query(evap_fabric, "variables")
+
+# trying to find available times before setting the variable results in an error
+# query(evap_fabric, "times")
+
+# only one variable, "et"
+variables(evap_fabric) <- "et"
+variables(evap_fabric)
+
+# now that the variable is set, we can explore available times
+query(evap_fabric, "times")
+```
+
+### Datasets not in GDP
+
+Any dataset available online that follows OPeNDAP protocol can be used with `geoknife`. These datasets can be found through web searches or other catalogs and require finding out the OPeNDAP endpoint (URL) for the dataset. This url is used as the input to the argument `url` in `webdata`. 
+
+We searched [NOAA's OPenDAP data catalog](https://opendap.co-ops.nos.noaa.gov/) and found this data from the Center for Operational Oceanographic Products and Services THREDDS server. It includes forecasts for water levels, water currents, water temperatures, and salinity levels for Delaware Bay. Since it is forecast data, the times associated with the data will change. To create a webdata object from this dataset, just use the OPeNDAP url. Then query variables and time as we did before.
+
+```{r create-fabric-NOAA, message=FALSE, warning=FALSE}
+DelBay_fabric <- webdata(url="http://opendap.co-ops.nos.noaa.gov/thredds/dodsC/DBOFS/fmrc/Aggregated_7_day_DBOFS_Fields_Forecast_best.ncd")
+query(DelBay_fabric, "variables")
+
+# need to set the variable(s) in order to query the times
+variables(DelBay_fabric) <- c("Vwind", "temp")
+query(DelBay_fabric, "times")
+```
+
+Here is a second example of using a non-GDP dataset. This data was found under the [data section on Unidata's website](http://www.unidata.ucar.edu/data/#home). This is aggregated UNIWISC satellite data for Earth's "surface skin" temperature. 
+
+```{r create-fabric-UNIWISC, message=FALSE, warning=FALSE}
+skinT_fabric <- webdata(url="http://thredds.ucar.edu/thredds/dodsC/satellite/SFC-T/SUPER-NATIONAL_1km")
+skinT_var <- query(skinT_fabric, "variables")
+
+# need to set the variable(s) in order to query the times
+variables(skinT_fabric) <- skinT_var
+query(skinT_fabric, "times") # your times might be different because this is forecast data
+```
+
+Both examples we've included here use aggregated data, meaning there is a different file for each year. Some data that you encounter might not be aggregated. In these cases, you will need to create more than one geojob and join data at the end.
```

Comment:
I think the definition of aggregated here should be that the URL represents all data of this type on the server vs. non-aggregated has many URLs to access the same data thus requiring multiple jobs.  Saying there is a different file for each year doesn't seem to convey that.